### ISSUE TITLE
Release SkipVAR artifacts on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [fakerone-li/SkipVAR#1]
Hi @fakerone-li 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work on Arxiv and was wondering whether you would like to submit it to [hf.co/papers](https://hf.co/papers) to improve its discoverability. If you are one of the authors, you can submit it at https://huggingface.co/papers/submit.

The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance), you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I have taken a look at your Github repository, and noticed that you released pre-trained decision models,
which are available under the "./SkipVAR" directory. It'd be great if you could host those on Hugging Face
instead, as this would give your models better visibility and discoverability. We can add tags in the model cards so that people find the models easier, link it to the paper page, etc.

In this case, we could leverage the [PyTorchModelHubMixin](https://huggingface.co/docs/huggingface_hub/package_reference/mixins#huggingface_hub.PyTorchModelHubMixin) class which adds `from_pretrained` and `push_to_hub` to any custom `nn.Module`. Alternatively, one can leverages the [hf_hub_download](https://huggingface.co/docs/huggingface_hub/en/guides/download#download-a-single-file) one-liner to download a checkpoint from the hub.

We encourage researchers to push each model checkpoint to a separate model repository, so that things like download stats also work. We can then also link the checkpoints to the paper page.

I have also seen that you released high-variation benchmark datasets, which are also
not available on Hugging Face. Would you be interested in hosting those on the hub as well?
Would be awesome to make the dataset available on 🤗 , so that people can do:

```python
from datasets import load_dataset

dataset = load_dataset("your-hf-org-or-username/your-dataset")
```
See here for a guide: https://huggingface.co/docs/datasets/loading.

Besides that, there's the [dataset viewer](https://huggingface.co/docs/hub/en/datasets-viewer) which allows people to quickly explore the first few rows of the data in the browser.

Let me know if you're interested/need any help regarding this!

Cheers,

Niels

**Comment by fakerone-li** [comment]
Hi Niels,
Thanks for reaching out! We’re already planning to release our models and datasets on Hugging Face Hub soon. We’ll definitely take advantage of the features you mentioned. Thanks again for the reminder!
Best,
Li
